app/controllers/admin/forms_controller.rb

```diff
@@ -237,11 +237,7 @@ def example
     end
 
     def js
-      if @form.legacy_form_embed
-        render(partial: 'components/widget/fba', formats: :js, locals: { form: @form })
-      else
-        render(partial: 'components/widget/fba2', formats: :js, locals: { form: @form })
-      end
+      render(partial: 'components/widget/fba2', formats: :js, locals: { form: @form })
```

Comment:
Suggest renaming the remaining file to `_fba`. fba2 without fba is confusing.

Reply:
good thought!

will do